webapp/app.py

```diff
@@ -57,4 +142,20 @@ def process_invoices_status_data(task_id: str) -> Response:
 
         return jsonify({"status": task_status, "logs": logs})
 
+    @app.route("/logout")
+    @login_required
+    def logout() -> str:
+        """Log out user by clearing Flask session and cookies.
+
+        Note: This doesn't log the user out of ALB or Touchstone.
+        """
+        # clear Flask session
```

Comment:
Nit picky: I might add we're clearing parsed OIDC data from session

Reply:
Let me know what you think of [the updated description](https://github.com/MITLibraries/alma-sapinvoices-ui/pull/60/commits/3072861e787c1c454a76871a4720a1de1730c271#diff-55f12a9f17b4ba61d3e37852150b7d78b8aedbe3629bc4ce8173d4b572297a54R153-R166). 🤔

Reply:
I dig it!  Thanks for updating.